client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -148,35 +152,74 @@ class TransferDomainPrecheck extends React.PureComponent {
-						'It might take a few minutes for any changes to take effect.',
-					{
-						components: {
-							a: (
-								<a
-									href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
-									rel="noopener noreferrer"
-									target="_blank"
-								/>
-							),
-						},
-					}
-				);
+		let heading = translate( 'Lock status unavailable.' );
+		if ( true === unlocked ) {
+			heading = translate( 'Domain is unlocked.' );
+		} else if ( false === unlocked ) {
+			heading = translate( 'Unlock the domain.' );
+		}
+
+		let message = translate( 'Your domain is unlocked at your current registrar.' );
+		if ( false === unlocked ) {
+			message = translate(
+				"Your domain is locked to prevent unauthorized transfers. You'll need to unlock " +
+					'it at your current domain provider before we can move it. {{a}}Here are instructions for unlocking it{{/a}}. ' +
+					'It might take a few minutes for any changes to take effect.',
+				{
+					components: {
+						a: (
+							<a
+								href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
+								rel="noopener noreferrer"
+								target="_blank"
+							/>
+						),
+					},
+				}
+			);
+		} else if ( null === unlocked ) {
+			message = translate(
+				"{{notice}}We couldn't get the lock status from your current registrar.{{/notice}} If you're sure it's unlocked then you " +
+					"can continue to the next step. If your domain isn't unlocked, then the transfer won't work. {{a}}Here are " +
+					'instructions to make sure your domain is unlocked.{{/a}}',
+				{
+					components: {
+						notice: <Notice showDismiss={ false } status="is-warning" />,
+						br: <br />,
+						a: (
+							<a
+								href={ support.INCOMING_DOMAIN_TRANSFER_PREPARE_UNLOCK }
+								rel="noopener noreferrer"
+								target="_blank"
+							/>
+						),
+					},
+				}
+			);
+		}
+
 		const buttonText = translate( "I've unlocked my domain" );
 
-		let lockStatusClasses = unlocked
-			? 'transfer-domain-step__lock-status transfer-domain-step__unlocked'
-			: 'transfer-domain-step__lock-status transfer-domain-step__locked';
+		let lockStatusClasses = 'transfer-domain-step__lock-status transfer-domain-step__unlocked';
+		if ( false === unlocked ) {
+			lockStatusClasses = 'transfer-domain-step__lock-status transfer-domain-step__locked';
+		} else if ( null === unlocked ) {
+			lockStatusClasses = 'transfer-domain-step__lock-status transfer-domain-step__unavailable';
+		}
+
+		let lockStatusIcon = 'checkmark';
+		if ( false === unlocked ) {
+			lockStatusIcon = 'cross';
+		} else if ( null === unlocked ) {
+			lockStatusIcon = 'info';
+		}
 
-		let lockStatusIcon = unlocked ? 'checkmark' : 'cross';
-		let lockStatusText = unlocked ? translate( 'Unlocked' ) : translate( 'Locked' );
+		let lockStatusText = 'Unlocked';
+		if ( false === unlocked ) {
+			lockStatusText = translate( 'Locked' );
+		} else if ( null === lockStatusText ) {
+			lockStatusText = translate( 'Unavailable' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Post unavailable' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).